docs/content/latest/yugabyte-cloud/cloud-network/cloud-vpc-intro.md

```diff
@@ -0,0 +1,75 @@
+---
+title: VPC network overview
+headerTitle: 
+linkTitle: Overview
+description: Requirements and considerations for setting up a VPC network.
+menu:
+  latest:
+    identifier: cloud-vpc-intro
+    parent: cloud-network
+    weight: 10
+isTocNested: true
+showAsideToc: true
+---
+
+A virtual private cloud (VPC) is a virtual network that you can define within a cloud provider. Once you create a VPC on a cloud provider, you can then connect it with other VPCs on the same provider. This is called peering. A VPC peering connection is a networking connection between two VPCs on the same cloud provider that enables you to route traffic between them privately, without traversing the public internet. VPC networks provide more secure connections between resources because the network is inaccessible from the public internet and other VPC networks. In addition, traffic in a VPC network doesn’t count against data transfer costs, as the traffic is all effectively local.
```

Comment:
> In addition, traffic in a VPC network doesn’t count against data transfer costs, as the traffic is all effectively local.
This is incorrect. You have the correct information on this page https://deploy-preview-10688--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-cloud/cloud-network/
"you won't have any Internet data transfer traffic. (Same region and cross region overages may still apply. Refer to Data transfer costs.)
"